docs/src/recipes/operating-on-dropped-elements.asciidoc

```diff
@@ -0,0 +1,27 @@
+////
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+////
+[[operating-on-dropped-elements]]
+== Operating on Dropped Elements
+
+One common scenario that happens when dropping elements using a traversal is the desire to perform some sort of operation on the elements being removed.  This simplest form of this scenario is returning the number of elements being dropped from the traversal, which is shown in the example below from the "modern" graph.
```

Comment:
"This simplest form of this scenario" - "The simplest"?

Reply:
Ack